src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -65,4 +65,20 @@ public static boolean isNonZeroUnsignedInteger(String s) {
             return false;
         }
     }
+
+    /**
+     * @@author baeldung-reused
```

Comment:
If I'm not wrong this is supposed to be `zihaooo9-reused`. I remember someone asking it in the forum before

Reply:
https://nus-cs2103-ay2122s1.github.io/website/schedule/week12/admin.html#policy-on-reuse